index.html

```diff
@@ -112,7 +112,7 @@
 		<div class="col-sm-2">Andr&eacute; Massey&nbsp;&nbsp;<i class="fa fa-rocket"></i></div>
 		<div class="col-sm-2">
 			<a href="https://github.com/DreTGiant" target="_blank"><i class="fa fa-github-square fa-2"></i></a> &nbsp;&nbsp;
-			<i class="fa fa-linkedin fa-2"></i> &nbsp;&nbsp;
+			<a href="https://www.linkedin.com/in/andremassey1" target="_blank"><i class="fa fa-linkedin fa-2"></i> &nbsp;&nbsp;
```

Comment:
Make sure to add a closing </a> tag after the closing </i> tag so that the whole icon (and nothing more) is a hyperlink. Please fix and then I'll merge the request.